stacks/java-openliberty-gradle/devfile.yaml

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2021 IBM Corporation and others
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+schemaVersion: 2.0.0
+metadata:
+  name: java-openliberty-gradle
+  version: 0.1.0
+  displayName: "Open Liberty Gradle"
+  description: Java application Gradle built stack using the Open Liberty runtime
```

Comment:
We're trying to make sure all of the stacks have icons associated with them. Could you add an icon for this stack? This is the one we're using for the other openliberty stack: https://github.com/devfile/registry/blob/main/stacks/java-openliberty/devfile.yaml#L23

Reply:
@johnmcollier Thanks for taking a look. I added the icon entry.